### PR TITLE
YWCA-3 Footer/Copyright

### PR DESCRIPTION
**Pull request by nanbul98**
Hardcoded version of footer completed.

### CODE REVIEW

**Comment by sumeetgrewal** [comment]
Please pull from master

File: style.css [hunk 1]
```diff
@@ -4,4 +4,29 @@ Author: Code The Change Foundation
...
+}
+.flex-container {
+  display: flex;
+  flex-direction: row;
+}
+.footer p:last-child {
+  margin-bottom: 10;
+  font-family: Gotham;
+}
+#vl {
```
Comment (sumeetgrewal) [inline style.css]: Please use a descriptive ID

File: style.css [hunk 2]
```diff
@@ -4,4 +4,29 @@ Author: Code The Change Foundation
...
+  position: absolute;
+  bottom: 0;
+  padding: 10px 10;
+  text-align: center;
+  font-family: Gotham;
+  font-size: 12;
+  background-color: #EEEEEE;
+  border-top: 10px solid #e5e5e5;
+}
+.flex-container {
```
[the remark this answers is not in the document]
Comment (nanbul98) [reply style.css]: flex-container is available in bootstrap 4, but it looks like we're using bootstrap 3 so i kept the flex-container as it is

File: functions.php [hunk 3]
```diff
@@ -14,6 +12,11 @@ function ywca_enqueue_styles() {
 add_action( 'wp_enqueue_scripts', 'bootstrap_enqueue_styles' );
 add_action( 'wp_enqueue_scripts', 'ywca_enqueue_styles' );
 
+
+
+add_action( 'wp_enqueue_scripts', 'bootstrapstarter_enqueue_styles' );
+add_action( 'wp_enqueue_scripts', 'bootstrapstarter_enqueue_scripts' );
+
```
Comment (sumeetgrewal) [inline functions.php]: Bootstrapstarter functions don't exist, use the enqueue functions from functions.php in master.

File: footer.php [hunk 4]
```diff
@@ -1,6 +1,15 @@
-
-<?php wp_footer() ?>
-
-</body> 
-
-</html>
\ No newline at end of file
+<link rel="stylesheet" href="/assets/css/ywca.css">
```
Comment (sumeetgrewal) [inline footer.php]: Remove stylesheet link here and fix functions.php enqueue scripts, that should import ywca.css

**Pull request merged**